plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -606,6 +613,49 @@ public String commit(final CommitParams params) throws IOException {
     @Override
     public void getEvents(final GetEventsParams params, MessageProcessor<Event> messageProcessor) {}
 
+    @Override
+    public void getContainerLogs(final GetContainerLogsParams params, MessageProcessor<LogMessage> containerLogsProcessor)
+            throws IOException {
+        String container = params.getContainer(); // container ID
+        Pod pod = getChePodByContainerId(container);
+        if (pod != null) {
+            String podName = pod.getMetadata().getName();
+            boolean[] ret = new boolean[1];
+            ret[0] = false;
+            try (LogWatch watchLog = openShiftClient.pods().inNamespace(openShiftCheProjectName).withName(podName)
```

Comment:
I think it might be cleaner to move the logs processing into a separate class. `OpenShiftConnector` is getting very long and complicated.